Abha_Rana/login_signup/userjwt/userjwt/DbModels/usersdbContext.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace userjwt.DbModels
+{
+    public partial class usersdbContext : DbContext
+    {
+        public usersdbContext()
+        {
+        }
+
+        public usersdbContext(DbContextOptions<usersdbContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Users> Users { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("Server=CYG292;Database=usersdb;Trusted_Connection=True;");
```

Comment:
connection string should not be present in db context file . It should be initalized once in startup class .

Reply:
connection string initialized in startup class and defined in appsettings.json